README.md

```diff
@@ -48,4 +55,13 @@ $ curl -X POST --url http://localhost:8001/plugins/ \
    --data 'config.sample_ratio=100' \
    --data 'config.service_name=kong'
    --data 'config.service_instance_name=kong-with-skywalking'
-```
\ No newline at end of file
+```
+
+# Contact Us
+* Submit an [issue](https://github.com/apache/skywalking/issues) with `[NIGNX-LUA]` as the issue title prefix.
```

Comment:
Wrong prefix

Reply:
what do you think about `KONG_LUA` as prefix?